projects/kit/internal/primitive-calendar-range/primitive-calendar-range.component.ts

```diff
@@ -99,6 +101,16 @@ export class TuiPrimitiveCalendarRangeComponent implements OnInit {
     monthOffset: TuiTypedMapper<[TuiMonth, number], TuiMonth> = (value, offset) =>
         value.append({month: offset});
 
+    ngOnChanges({
+        defaultViewedMonthFirst,
+        defaultViewedMonthSecond,
+    }: SimpleChanges): void {
+        this.updateViewedMonths(
+            defaultViewedMonthFirst?.currentValue,
```

Comment:
If you only need current value you can just use the values as is, no need for `SimpleChanges`

Reply:
No, we need to check if it comes new value of `defaultViewedMonth`